src/providers/postgres/qgspostgresconn.h

```diff
@@ -183,6 +183,12 @@ class QgsPostgresConn : public QObject
     Q_OBJECT
 
   public:
+    /// @param shared allow using a shared connection. Should never be
```

Comment:
Is `///` picked up by doxygen?

For multiline doxygen comments it's common to use `/**` throughout the sourcecode.

Shouldn't it be deprecated and then removed? It seems to me that adding a TODO for deprecation is an unrequired extrastep.